var/spack/repos/builtin/packages/git/package.py

```diff
@@ -69,6 +76,7 @@ class Git(Package):
 
     def install(self, spec, prefix):
         env['LDFLAGS'] = "-L%s" % spec['gettext'].prefix.lib + " -lintl"
+        env['NO_GETTEXT'] = "YesPlease"
```

Comment:
This goes back to when there was a bug with gettext in spack and I failed to notice that when I created the PR, sorry